docs/data/material/experimental-api/css-theme-variables/migration.md

```diff
@@ -0,0 +1,295 @@
+# Migrating to CSS variables
+
+<p class="description">A step-by-step migration guide to start using CSS theme variables in your project.</p>
+
+This is a guide for an existing Material UI project to migrate to CSS theme variables. The migration aims to fix the dark mode flickering issue.
+
+## 1. Use the new provider
+
+### Without a custom theme
+
+If you haven't used [`ThemeProvider`](/material-ui/customization/theming/#theme-provider), simply wrap you application with the new provider:
```

Comment:
```suggestion
If you aren't using [`ThemeProvider`](/material-ui/customization/theming/#theme-provider), then all you need to do is wrap your application with the CSS variables provider:
```

Reply:
> with the CSS variables provider:

What do you think about making it more explicit with

```md
with the CSS theme variables provider:
```

or 

```md
with the `CssVarsProvider`:
```

Reply:
Good call—either of those sound good to me. 👍